pipelines/HyperSuprimeCam/ProcessCcd.yaml

```diff
@@ -10,6 +10,11 @@ tasks:
       file: $AP_PIPE_DIR/config/HSC/characterizeImage.py
       # Do not integrate file and pipeline configs until obs config files are
       # gone, to make it easier to check for changes on the obs side.
+      # Config file wipes out all pre-existing configs, so copy base pipeline
+      # config on top.
```

Comment:
Because of this comment, does it mean we have to copy all configs from the base processCcd.yaml? Or has that already been done? Could you work around it by doing a `file:ProcessCcd.yaml` above here?

Reply:
No, there's no such thing as importing a pipeline into a task config. There's a global pipeline import, but the `characterizeImage.py` gets applied after it.